addon/session-stores/cookie.js

```diff
@@ -76,25 +77,51 @@ export default BaseStore.extend({
   */
   cookieExpirationTime: null,
 
-  _secureCookies: window.location.protocol === 'https:',
+  _cookies: computed(function() {
+    let owner = getOwner(this);
+
+    return owner.lookup('service:cookies');
+  }),
```

Comment:
This can be injected as a normal service as ember-cookies will be a dependency of ember-simple-auth and thus always be present.